cpp/tests/utilities/column_utilities.cu

```diff
@@ -136,6 +137,19 @@ void expect_equal_buffers(void const* lhs, void const* rhs,
                             typed_rhs));
 }
 
+// copy column bitmask to host (used by to_host())
+std::vector<bitmask_type> bitmask_to_host(cudf::column_view const& c) {
+  std::vector<bitmask_type> host_bitmask;
+  if (c.nullable()) {
+    auto num_bitmasks = bitmask_allocation_size_bytes(c.size()) / sizeof(bitmask_type);
```

Comment:
Is there a reason not to use num_bitmask_words here?  I don't think there's any value in bringing back the empty padding.

Reply:
One of the UTILITIES_TESTS fails if I do that.  I'll look at why today.

Reply:
I looked, and since `make_null_mask_vector`, used by `column_utilities_tests.cu`, uses exactly this arithmetic, I need to use exactly this arithmetic to get that test comparison to pass.

Besides, I think that if you were to copy the bitmask to host, and then create a new device bitmask from a direct copy of it back to the device, correctness requires copying the whole thing. I think the copy overhead is negligible.